schemas/mqtt/journey.json

```diff
@@ -181,6 +181,16 @@
                     "maximum": 180.0000
                   }
                 }
+              },
+              "description": {
+                "$id": "#/properties/route/properties/stopPlaces/items/properties/description",
+                "type": "string",
+                "description": "Description for the stop"
```

Comment:
In EnTur i see that that the field is described a "Description for travelers" an Sean is always  right :D.